experimental/packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -301,15 +300,14 @@ export abstract class InstrumentationBase<T = any>
 
 function isSupported(
   supportedVersions: string[],
-  version?: string,
-  includePrerelease?: boolean
+  version?: string
 ): boolean {
   if (typeof version === 'undefined') {
     // If we don't have the version, accept the wildcard case only
     return supportedVersions.includes('*');
   }
 
   return supportedVersions.some(supportedVersion => {
-    return satisfies(version, supportedVersion, { includePrerelease });
+    return supportedVersion === '*' || satisfies(version, supportedVersion);
```

Comment:
Hmm, looks like this is not the same behavior anymore. In the gRPC instrumentation, we set the supported version to `1.*`, but this now evaluates to `false` with any 1.x.x version (which is why the tests fail).

Reply:
this may affect lot's of third-party instrumentations and instrumentations over at https://github.com/open-telemetry/opentelemetry-js-contrib

Reply:
created: https://github.com/omichelsen/compare-versions/issues/70

Reply:
Hey, currently looking at this! Thanks for creating that :) 

I see a few community packages doing this too

* [opentelemetry-instrumentation-mysql](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-mysql)
* [opentelemetry-instrumentation-pg](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-pg)

This package also just specifies 1 as the version? 
* [opentelemetry-instrumentation-router](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-router)

Reply:
I tested out adding a ^ in front of the version in the instrumentation-grpc package and that does seem to resolve this issue but I have listened to too many of Linus's rants to think shipping this satisfies behaviour change as it stands is a good idea :laughing:

Reply:
I have opened a PR to fix this on compare-versions https://github.com/omichelsen/compare-versions/pull/71

I will see where we get too